themes/radial/entries.typ

```diff
@@ -1,49 +1,106 @@
 #import "./colors.typ": *
 #import "./icons/icons.typ"
-#import "./components/title.typ" as components
+#import "./components/components.typ"
+#import "./components/title.typ": *
 #import "/utils.typ"
 #import "./metadata.typ": entry-type-metadata
 
 // TODO: make an actual cover
-#let cover(ctx: (:)) = [
-  #set align(center)
-  *Radial Cover*
-]
+#let cover(
+  ctx: (:),
+) = {
+    import components: label
+
+    align(center + horizon)[
+      #text(size: 24pt, font: "Calibri")[
+          Radial Theme
+        ]
+      ]
+        place(dx: 90pt, dy: -340pt, components.label("identify", size: 4.7em))
```

Comment:
Because you imported `label` you can refer to it directly, instead of by `components.label`